### ISSUE TITLE
Edit history changes

### ISSUE THREAD

**Issue by edwardchalstrey1** [colouring-cities/colouring-london#822]
-  [ ] Rename "History" to "Edit History"
- [ ] Add a "last edited by" which is just the top result  from the Edit history page to the main edit page at the bottom (for all categories)
- [ ] Add an icon for the edit method in the Edit history that distinguishes between manual and automated
- [ ] For the automated edits, it should read "Last edited by [method]" as oppose to "Last edited by [person]"

**Comment by edwardchalstrey1** [comment]
PH: I think ideally we should in fact alter the system so that if the current edit is overwritten with a new edit that matches an historical one, then the old verifications should also be restored with one extra verification also added for the original editor's entry.

**Comment by tomalrussell** [comment]
> if the current edit is overwritten with a new edit that matches an historical one, then the old verifications should also be restored 

Verifications are stored independently of edits, and this already works.

If any number of users verify that an attribute should have a particular value (e.g. a building was built in the year 2000) then that count will be shown when the current state of that attribute is equal to the verified value. 

For example:
- one user edits a building to set build year to 2000
- five users verify this; the verification count is 5
- another user changes the build year to 2010; the verification count for 2010 is 0
- the user changes the build year back to 2000; the verification count is 5 again

- at any point, a user who has verified a particular attribute value can see what value they verified, and can remove their verification if necessary
- at any point, a user who has not verified a particular attribute yet (or has removed any previous verification) can verify the current value, which will increase the verification count by one

> one extra verification also added for the original editor's entry

I don't think this should be implemented - verification is independent of data entry. If an editor wants to verify their own data entry, they can, but we shouldn't be automatically adding verifications.

**Comment by edwardchalstrey1** [comment]
Thanks @tomalrussell 

> the user changes the build year back to 2000; the verification count is 5 again

Does this assume that no-one clicked "remove"? It looked to be like when that is clicked, we delete the row in the `building_verification` table. Or perhaps I'm misunderstanding how the verification count is handled?

**Comment by tomalrussell** [comment]
Yes, if anyone clicks remove that would affect their own verification - so could reduce the total count by at most one.

See https://github.com/colouring-cities/colouring-london/blob/f5ee154b5dcc2c691af78139f07aaf68dfd08d9a/app/src/api/dataAccess/verify.ts#L109-L114

**Comment by polly64** [comment]
@mdsimpson42 can you check these ones are done?
